video/out/gpu/video.c

```diff
@@ -448,6 +448,8 @@ const struct m_sub_options gl_video_conf = {
         {"deband", OPT_FLAG(deband)},
         {"deband", OPT_SUBSTRUCT(deband_opts, deband_conf)},
         {"sharpen", OPT_FLOAT(unsharp)},
+        {"clear-framebuffer", OPT_CHOICE(clear_framebuffer,
+            {"conditional", 0}, {"always", 1}, {"never", -1})},
```

Comment:
"always" should be the default then.

Reply:
Also, "never" actually makes no sense, because it'll display old framebuffer contents.

Reply:
my intend was to make it easier for me to debug user problems related to rendering or that specific bug. i could just tell them to test with 'never' and if it fixes their issue i can just direct them upstream to apple. telling them to build mpv themselves and commenting out the clearing code is usually too hard for most of them.

if this is not enough for it to exist, fine by me?

Reply:
int he case of removal i would just make a OPT_BOOL out of it